src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2088,55 +2087,66 @@
             </attribute>
         </optional>
     </define>
-    
+
     <define name="poem">
-        <!-- Use: poem is a complete poem or fragment thereof. -->
         <a:documentation>  Use: poem is a complete poem or fragment thereof.  </a:documentation>
-        <element name="section">
-            <!-- HTML content model: (a | abbr | address | article | aside | audio | b | bdi | bdo | blockquote | br | button | canvas | cite | code | data | datalist | del | details | dfn | dialog | div | dl | em | embed | fieldset | figure | footer | form | h1 | h2 | h3 | h4 | h5 | h6 | header | hr | i | iframe | img | input | ins | kbd | keygen | label | link | main | map | mark | math | menu | meta | meter | nav | noscript | object | ol | output | p | pre | progress | q | ruby | s | samp | script | section | select | small | span | strong | style | sub | sup | svg | table | template | textarea | time | u | ul | var | video | wbr | (text))* -->
-            <!-- Strict content model: (h1 | span | p | section | img | figure)* -->
-            <ref name="attlist.poem"/> <!-- @epub:type, @class, @id, @title, @xml:space, @xml:lang, @lang, @dir -->
-            <zeroOrMore>
+        <element name="div">
+            <attribute name="class">
+                <value>verse</value>
+            </attribute>
+            <optional>
                 <choice>
                     <ref name="title"/> <!-- h1, h2, h3, h4, h5, h6 -->
-                    <ref name="author.block"/> <!-- p -->
-                    <ref name="hd"/> <!-- h1, h2, h3, h4, h5, h6 -->
-                    <ref name="dateline"/> <!-- span -->
-                    <ref name="epigraph"/> <!-- blockquote, p -->
-                    <ref name="byline"/> <!-- span -->
-                    <ref name="linegroup"/> <!-- div, section -->
-                    <ref name="line"/> <!-- p -->
-                    <ref name="pagebreak.block"/> <!-- div -->
-                    <ref name="img"/> <!-- img -->
-                    <ref name="figure.imggroup"/> <!-- figure -->
-                    <ref name="sidebar"/> <!-- aside, figure -->
+                    <ref name="bridgehead"/>
                 </choice>
-            </zeroOrMore>
+            </optional>
+            <ref name="poem.linegroup"/>
+            <optional>
+                <element name="p">
+                    <attribute name="class">
+                        <value>verse-author</value>
+                    </attribute>
+                </element>
+            </optional>
         </element>
     </define>
 
-    <define name="attlist.poem" combine="interleave">
-        <attribute name="epub:type">
-            <list>
-                <ref name="types"/>
+    <define name="poem.linegroup">
+        <element name="p">
+            <attribute name="class">
+                <value>linegroup</value>
+            </attribute>
+            <oneOrMore>
```

Comment:
This allows a line to contain only a line number. It is also possible for some lines to have line numbers, and some lines to not have line numbers. It's a bit relaxed, but this is also how it was handled in DTBook where this structure stems from, and the new guidelines doesn't put any restrictions on it either.

@martinpub @AndersEkl does the following look like it should be allowed?

```html
<div class="poem">
    <p class="linegroup">
        <span class="linenum">13</span>
    </p>
    <p class="linegroup">
        <span class="linenum">1</span><span class="line">Lorem ipsum</span><br/>
        <span class="line">Lorem ipsum</span><br/>
        <span class="line">Lorem ipsum</span><br/>
        <span class="linenum">4</span>
        <span class="linenum">3</span><span class="line">Lorem ipsum</span><br/>
        <span class="linenum">3</span>
    </p>
</div>
```

Reply:
I can definitely see cases where there are lines containing only a line number. That should be allowed. I am not sure it is necessary to put too much effort into building rules about this. Speaking from my own perspective, we rarely use line numbers at all. I am fine with having a rather relaxed validation here. I am not sure if @martinpub has a different opinion.